### PR TITLE
remove reflect in grpc server

### PR DESCRIPTION
**Pull request by gaoxinge**
**What this PR does**:

- remove reflect for `SetProxyImpl` in grpc server
- update protobuf

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by zouyx — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

File: protocol/grpc/server.go [hunk 1]
```diff
@@ -134,18 +136,12 @@ func waitGrpcExporter(providerServices map[string]*config.ServiceConfig) {
 func registerService(providerServices map[string]*config.ServiceConfig, server *grpc.Server) {
 	for key, providerService := range providerServices {
 		service := config.GetProviderService(key)
-
 		ds, ok := service.(DubboGrpcService)
 		if !ok {
```
Comment (hxmhlt) [inline protocol/grpc/server.go:140]: Is 'ds' may be nil ?

**Review by hxmhlt — COMMENTED** [review]
LGTM

File: protocol/grpc/server.go (hunk 1 above)
Comment (hxmhlt) [reply protocol/grpc/server.go:140]: > Is 'service' may be nil ?

Comment (gaoxinge) [reply protocol/grpc/server.go:140]: `service` check exist: https://github.com/apache/dubbo-go/blob/master/config/config_loader.go#L259-L264.

**Pull request merged**